ReactAndroid/src/main/java/com/facebook/react/views/webview/ReactWebViewManager.java

```diff
@@ -277,6 +278,11 @@ public void setUserAgent(WebView view, @Nullable String userAgent) {
     }
   }
 
+  @ReactProp(name = "mediaPlaybackRequiresUserAction")
+  public void setMediaPlaybackRequiresUserAction(WebView view, boolean requires) {
+    view.getSettings().setMediaPlaybackRequiresUserGesture(requires);
```

Comment:
Ah now I see what you meant by those established instances. OK you've convinced me :)